<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 95, 180);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Tlwg Typo</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 24pt &quot;Tlwg Typo&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>000000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BIMO">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>37</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(46, 194, 126);</string>
    </property>
    <property name="text">
     <string>BMO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>541</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="Button7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="ButtonMM">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>M-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Button4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Button3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="ButtonM">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Button6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="ButtonMult">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="Button9">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="ButtonMP">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>M+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="ButtonAdd">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="ButtonDiv">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Button2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Button5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Button8">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Button1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="ButtonAC">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Button0">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="ButtonPM">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 194, 126);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(26, 95, 180);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="ButtonSub">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="ButtonEq">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(28, 113, 216);
font: 500 24pt &quot;Ubuntu&quot;;
color: rgb(143, 240, 164);
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: white;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
